src/tasks_io.yaml

```diff
@@ -8,5 +8,8 @@ automatic_scaling:
   max_idle_instances: 1
 
 handlers:
+  - url: /_ah/queue/deferred.*
```

Comment:
- [ ] Test `/_ah/queue/deferred.*` endpoints - I'm pretty sure they're unsupported. I think only `/_ah/queue/deferred` will work with the new bundled solution, since Flask doesn't have the idea of regex routing. We might be able to patch it though.

Reply:
I also don't know that we necessarily need this?